src/acl.js

```diff
@@ -0,0 +1,53 @@
+class ACL {
+  constructor (keystore, key, keys = []) {
+    this._keystore = keystore
+    this._key = key
+    this._keys = keys && Array.isArray(keys) ? keys : keys ? [keys] : []
+  }
+
+  getSigningKey () {
+    return this._key
+  }
+
+  getPublicSigningKey (format = 'hex') {
```

Comment:
I think `ipfs-log` always uses this format. Is it going to change or can we assume it'll always be 'hex'?